src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -994,24 +968,10 @@ def merge_corpus(self):
       del os.environ[constants.STDERR_FILENAME_ENV_VAR]
     except KeyError:
       pass
+
     self._executable_path = self.afl_showmap_path
-    # Hack around minijail.
-    showmap_args = self._fuzz_args
-    showmap_args[-1] = '1'
+    showmap_args = self.generate_afl_args(use_showmap=True)
     # Remove arguments for afl-fuzz.
-    self.remove_arg(showmap_args, constants.INPUT_FLAG)
-    self.remove_arg(showmap_args, constants.DICT_FLAG)
```

Comment:
i dont see DICT_FLAG and SKIP_DETERMINISTIC_FLAG removed in new code in generate_afl_args ?